include/logging/log_core.h

```diff
@@ -407,6 +407,14 @@ static inline char z_log_minimal_level_to_char(int level)
 extern struct log_source_const_data __log_const_start[0];
 extern struct log_source_const_data __log_const_end[0];
 
+/**@brief Enum with possible actions for strdup operation.
```

Comment:
add space before brief. comment can be closed in the same line.

Reply:
still missing space.

Reply:
fixed